src/Bubble.js

```diff
@@ -52,6 +53,21 @@ export default class Bubble extends React.Component {
     return null;
   }
 
+  renderTicks() {
+    if (this.props.currentMessage.sent || this.props.currentMessage.received) {
+      const {currentMessage} = this.props;
+      if (this.props.renderTicks) {
```

Comment:
This could go above the first `if`
The user might want to have different props for his sent / received props